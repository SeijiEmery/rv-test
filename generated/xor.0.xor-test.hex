0000000    33  c2  41  00  b3  c2  51  00  b3  c3  71  00  33  c4  81  00
0000020    b3  c4  52  00  73  00  10  00                                
0000030
