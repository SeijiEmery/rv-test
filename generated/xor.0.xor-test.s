xor     x4, x3, x4
xor     x5, x3, x5

xor     x7, x3, x7
xor 	x8, x3, x8
xor		x9, x5, x5
nop
nop
nop
nop
nop
ebreak
