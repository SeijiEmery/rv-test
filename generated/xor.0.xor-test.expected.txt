R3 = 0
R4 = 0
R5 = 1
R7 = 2147483647
R8 = 2147483648
R9 = 0
